.bandit.yml

```diff
@@ -3,3 +3,4 @@ skips: []
 # No need to check for security issues in the test scripts!
```

Comment:
This file and `.flake8` should have been deleted correct?

Reply:
That wasn't done as part of the drift manager. It was the first one that I touched the first round/